code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -136,12 +136,22 @@
 	var/vine_grab_distance = 5
 	/// Whether or not this plant is ghost possessable
 	var/playable_plant = TRUE
-
 	discovery_points = 2000
 
+/mob/living/simple_animal/hostile/venus_human_trap/Initialize()
+	remove_verb(/mob/living/verb/pulled) //No pulling people into the vines
```

Comment:
Does this not still allow for using ctrl-click?

Reply:
It does not, ctrl-click does not work anymore with this verb removed.